homework2/homework2.js

```diff
@@ -0,0 +1,129 @@
+//----------------------- EX1 -----------------------
+/*Creați o funcție care primește ca și parametru un număr și returnează dublul său dacă numărul primit este impar, 
+altfel returnează numărul scăzut cu 2 dacă este par. Afișați rezultatul în consolă.*/
+
+function exercise1(nr) {
+    if (nr % 2 != 0) {
+        return nr * 2;
+    } else if (nr % 2 == 0) {
```

Comment:
You should use `===` instead of `==`

Reply:
I used `===`, then I thought, in which cases it wouldn't be the same type of data? and I tested it on string numbers. Apparently, the remainder operator automatically converts the string to number and this is why I didn't use the strict equality. But you're right, it's cleaner with it.

Reply:
This is just a simple example. 
Imagine if you had to validate codes made up of numbers, and in fact the user sends a string that can be a regex and this type of code will pass `==` and then he can access sensitive data with a simple hack 👍 This is why it is better to use `===` instead of `==`